src/usr/local/www/wizards/setup_wizard.xml

```diff
@@ -588,7 +588,7 @@
 	</fields>
 	<stepsubmitphpaction>
 		<![CDATA[
-		if (empty($_POST['lanipaddress']) || !is_ipaddr($_POST['lanipaddress'])) {
+		if (empty($_POST['lanipaddress']) || !is_ipaddr($_POST['lanipaddress']) || strcasecmp($_POST['lanipaddress'],"dhcp")!=0) {
```

Comment:
PHP isn't a language I use much, so maybe you can clarify. My concern was that the comparison lanipaddress != "dhcp" will be true every time lanipaddress is a numerical IP. if that's not the case, would strcasecmp work?